spec/service_consumers/pact_helper.rb

```diff
@@ -681,5 +681,16 @@ def url_encode(str)
         target_content_id: primary_publishing_organisation.content_id,
       )
     end
+
+    provider_state "a published content item exists with content_id: 19ad249e-7ac4-4aa4-8ab4-b6c5f381c043" do
```

Comment:
**question:** pertinent field

Is the content ID what we'll care about wherever we need to use this, or will it be the base path?

Reply:
We do have [other examples that use base paths in their name](https://github.com/alphagov/publishing-api/pull/2950/files#diff-7d17ffbce13a8dc8820ff122a86f9a63284e76991d64a9981a7eb7c5697f6bfcR620-R638). I guess I was thinking this could be "a published content item exists with base_path /my-document"